admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -67,31 +67,55 @@ public ReplicationUtils(
 
   // mutator methods
   public ReplicationSiteField createSite(
-      String name, AddressField address, @Nullable String rootContext) {
+      String name, AddressField address, @Nullable String rootContext, boolean isDisableLocal) {
     String urlString = addressFieldToUrlString(address, rootContext);
     ReplicationSite newSite = siteManager.createSite(name, urlString);
+    newSite.setIsDisabledLocal(isDisableLocal);
     siteManager.save(newSite);
 
     return getSiteFieldForSite(newSite);
   }
 
-  public boolean siteExists(String name) {
+  public boolean isDuplicateSiteName(String name) {
     return siteManager.objects().map(ReplicationSite::getName).anyMatch(name::equalsIgnoreCase);
   }
 
+  public ReplicationSite getSite(String id) {
+    try {
+      return siteManager.get(id);
+    } catch (NotFoundException e) {
+      return null;
+    }
+  }
+
+  public boolean isNotUpdatedSiteName(String id, String name) {
+    ReplicationSite site = this.getSite(id);
```

Comment:
✏️ [Nit] You could make it final and clarify for all that the reference will not changed in the method after.